### ISSUE TITLE
`cf spaces` does not err when authentication expired

### ISSUE THREAD

**Issue by DanielJonesEB** [cloudfoundry/cli#1051]
## Command  

`cf spaces`

## What occurred

```
$ cf spaces
Getting spaces in org engineerbetter as testing@engineerbetter.com...

name
No spaces found
```

## What you expected to occur

Something similar to when one does `cf orgs`:

```
$ cf orgs
Getting orgs as testing@engineerbetter.com...

FAILED
Authentication has expired.  Please log back in to re-authenticate.

TIP: Use `cf login -a <endpoint> -u <user> -o <org> -s <space>` to log back in and re-authenticate.
```

## CLI Version  

cf version 6.22.2+a95e24c-2016-10-27

## CC API Endpoint Version  

https://api.run.pivotal.io (API version: 2.68.0)

## Platform & Shell Details  

macOS Sierra, iTerm 2

**Comment by dkoper** [comment]
Thanks @DanielJonesEB! We'll take a look.

**Comment by dkoper** [comment]
We have addressed this in the 6.32.0 release.
